### DISCUSSION TITLE
Not able to upload javascript file in backend

### DISCUSSION THREAD

**Discussion by erwindegrave** [globaleaks/globaleaks-whistleblowing-software#3416] [General]
Hi everyone,

In the Theme customization in the admin panel, as mentioned in the user guide, I am trying to upload a Javascript file using the 'Files' button, but nothing happens.
I can upload images and pdfs, but not Javascript files.

What do I need to do to solve this issue?

Thanks.

**Comment by evilaliv3** [comment]
Hello,

It is currently not possibly to upload JavaScript files via the web
interface.

You can manually load them via the web interface. You should call them with
the number of the tenant (site) E.g. /var/globaleaks/scripts/1 is the
script file that will be loaded on the first tenant

**Reply by joir7** [reply]
Good morning evilaliv3

First of all, thanks in advance. Does the file have to be named in any specific way?

I've tried to leave a javascript in /var/globaleaks/scripts calling it in several ways, but I can't get it to work.

**Comment by evilaliv3** [comment]
Hello @joir7 

The file should be called with the number of the tenant.
E.g. /var/globaleaks/scripts/1 for tenant 1

**Comment by Jorge-CB** [comment]
Hello,

Pardon my ignorance and thanks in advance, I am new to using Globaleaks.

I want to change the structure of the footer but I have copied a file called '1.js' into the /var/globaleaks/scripts/ directory and I still haven't managed to change the footer.

The content of the 1.js test file is very simple:

![image](https://user-images.githubusercontent.com/132654228/236440647-1aba75e8-9f98-4ef8-8128-a204324f43b5.png)


I don't know what a tenant is. Is a tenant a site? 

I only have one site. Attached image:


![image](https://user-images.githubusercontent.com/132654228/236433066-cda590da-f5c6-46dd-a5ea-65d6312feecf.png)

Any help will be welcome.

Thanks.

**Comment by evilaliv3** [comment]
@Jorge-CB : the file should be called just "1" withouth ".js" at the end

**Comment by Jorge-CB** [comment]
Thank you very much! It works now.
